### ISSUE TITLE
Change event bug still persists in latest version when initializing pikaday

### ISSUE THREAD

**Issue by martinbarilik** [Pikaday/Pikaday#910]
This is duplicate from #149 . 

The bug still persists, as you can see in filddle https://jsfiddle.net/mArtinko5MB/2hd7nkvL/11/

All my date inputs are already filled with values and every of them has change event listener for noticing changes on form so current state is unworkable

**Comment by ConnectGrid** [comment]
Agreed. Being bitten by this bug, too. I can't believe it's persisted for 6 years without any movement. If the date field already has the same date value as assigned by Pikaday, then no `change` event should be fired since nothing has "changed".

**Comment by ConnectGrid** [comment]
I've solved this with a three-step solution.

1. Assign a `change` event listener ot the date field. Call `Event.stopPropagation()` within this event handler.
2. Initialize Pikaday for that field.
3. Remove the event handler assigned in Step 1.

This gives us the behavior that we expect.

**Comment by martinbarilik** [comment]
I am forced to replace Pikaday with something that does not fire change listener this way.

I tried to monkey patched it but with design i have it's stupidly non-logical to append so much code just for this bug. I wonder why nobody urges this to be fixed ASAP. I feel like i am missing something or every competent person doesn't care enough to even write something.
